### PR TITLE
Create cross-platform_pt 2

### PR DESCRIPTION
**Pull request by kaustubhavarma**
Part 2 of the work on cross platform abuse to upload on the website

### CODE REVIEW

File: uli-website/src/pages/blog/cross-platform_pt 2 [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+project: " "
+date: 17-04-2024
+---
+
+import ContentPageShell from "../../components/molecules/ContentPageShell.jsx"
+
+<ContentPageShell>
+
+We're continuing by taking a look at how cross-platform abuse operates on federated and centralised platforms, and the kind of solutions they must test and implement to tackle cross-platform abuse. 
+The most popular decentralised/federated social media platform is Mastadon; Threads and Bluesky have also followed suit into adopting federation protocols for social media platforms.
```
Comment (dennyabrain) [inline uli-website/src/pages/blog/cross-platform_pt 2:14]: typo in Mastodon

Comment (kaustubhavarma) [reply uli-website/src/pages/blog/cross-platform_pt 2:14]: oh! fixing it sorry

File: uli-website/src/pages/blog/cross-platform_pt 2 [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+##Federated Moderation
+
+On federated platforms mentioned above, it has been noted that hateful material can rapidly disseminate from one instance to [another](https://arxiv.org/pdf/2302.05915.pdf). 
+Federation policies help administrators of instances create rules that ban or modify content from [instances](https://arxiv.org/pdf/2302.05915.pdf).
+Administrators determine the rules for a particular instance, as opposed to traditional social media platforms which are more centralised, hire human moderators and have access to a vast range of data to refine their automated [classifiers](https://arxiv.org/pdf/2204.12709.pdf). 
+They do not have access to as many resources, and oversee only their own instance. Administrators of instances appear to be overwhelmed when it comes to moderation, with a gap observed in the time taken to respond to content that violates the policy of an [instance](https://arxiv.org/pdf/2302.05915.pdf). 
+In the decentralised platform, there are two levels of interaction: within instances, and across instances, and moderation therefore should also operate on these two levels as well.
```
Comment (dennyabrain) [inline uli-website/src/pages/blog/cross-platform_pt 2:24]: this is a great line.

Comment (dennyabrain) [reply uli-website/src/pages/blog/cross-platform_pt 2:24]: should aid comprehension for those who are new to this.

**Pull request merged**